### PR TITLE
Better upgrade documentation about the ``project.yaml.mako`` file.

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/upgrade_application.rst [hunk 1]
```diff
@@ -119,22 +132,26 @@ Create a ``project.yaml.mako`` file that contains:
...
-Where ``<folder>`` is the last element of the folder e.g. for
-``/home/user/c2cgeoportal`` it will be ``c2cgeoportal``,
+    project_folder: <project_folder>
+    project_package: ${package}
+    host: <host>
+    checker_path: /${instanceid}/wsgi/check_collector?
+    template_vars:
+        package: ${package}
+        srid: ${srid}
+        extent: <aplication_extent>
```
Comment (ochriste) [inline doc/integrator/upgrade_application.rst]: app
idem ci-après

File: doc/integrator/upgrade_application.rst [hunk 2]
```diff
@@ -29,17 +29,30 @@ Where ``<makefile>`` is your user make file (``<user>.mk``).
 Easy upgrading an application
 -----------------------------
 
-Verify that you have in your ``project.yaml.mako`` file the following ``template_vars``: ``package``, ``srid``, ``extent``, ``mobile_application_title``, ``apache_vhost``, for example:
+Verify that you have in your ``project.yaml.mako`` file like the following:
```
Comment (ochriste) [inline doc/integrator/upgrade_application.rst]: Verify that your ``project.yaml.mako`` file is like the following:
ou bien
Verify that you have in your ``project.yaml.mako`` file the following content:

**Comment by ochriste** [comment]
some details, otherwise LGTM

**Comment by sbrunner** [comment]
Thanks @ochriste fixed :-)

**Pull request merged**